### DISCUSSION TITLE
为什么使用PP-OCRv4检测、识别模型，检测识别出来的单词是连在一起的，不能区分每个单词。

### DISCUSSION THREAD

**Discussion by 1424153694** [PaddlePaddle/PaddleOCR#13203] [Q&A]
#### 问题描述 / Problem Description
为什么使用PP-OCRv4检测、识别模型，检测识别出来的单词是连在一起的，检测模型不是对每个单词进行检测画框，而是把整个短语句子放在一个检测框里，识别模型也无法识别出空格，导致模型识别出的单词内容是连在一起的，不能区分每个单词。

效果图:
![page_2_image_1](https://github.com/PaddlePaddle/PaddleOCR/assets/76905904/d59c41dd-e2be-48da-a02c-77f3de48cb32)

**Comment by lucasjinreal** [comment]
同样的问题，别问，问就是百度做的东西压根不考虑鲁棒场景。 不过你这个可能是英文模型，如果是中英文夹杂，好像padle严格不支持中英文合在一起的时候英文识别空格

**Comment by GreatV** [comment]
检测是按行检测的

![result](https://github.com/PaddlePaddle/PaddleOCR/assets/17264618/1ca038ac-2e54-48d8-8d2f-9ae83044c945)
